tests/test_get_h.py

```diff
@@ -113,7 +116,7 @@ def test_xrange():
     assert x1[1] < x2[1]
 
 
-def test_G():
+def test_g():
```

Comment:
Wasn't that renamed to `final_term_amplitude`?

Reply:
Yup. Fixed.